c2rust-analyze/src/main.rs

```diff
@@ -759,6 +778,7 @@ fn run(tcx: TyCtxt) {
                     &acx,
                     &info.dataflow,
                     &mut asn.perms_mut(),
+                    &mut maps,
```

Comment:
I don't think it's a good idea to share a single `AtomMaps` across the borrowck invocations for all functions.  It lets the map grow quite large and is unnecessary because different functions never interact with each other's atoms.  (This is similar to the reason we have separate `LocalPointerTable`s and `GlobalPointerTable`s for working with `PointerId`s.)  If (after investigating Polonius's `'static` handling) it's actually necessary for all functions to share the global/static part of the `AtomMaps`, I think a better way to go about it is to pass `maps.clone()` by value into each function's `borrowck` invocation, so they no longer share one big map.

Reply:
How would I determine in the `nll` output that polonius is having all function share the global/static part of the `AtomMaps`?